lib/hsa/mcwamp_hsa.cpp

```diff
@@ -4600,57 +4704,11 @@ HSADispatch::setLaunchConfiguration(int dims, size_t *globalDims, size_t *localD
     aql.group_segment_size   = kernel->static_group_segment_size + dynamicGroupSize;;
```

Comment:
extra semi colon, also would it be useful to print `kernel->private_segment_size` in debugging mode too?